api/src/main/java/org/openmrs/module/fhir/api/helper/FHIRClientHelper.java

```diff
@@ -58,8 +58,14 @@
 		CATEGORY_MAP.put(CATEGORY_ALLERGY, AllergyIntolerance.class);
 	}
 
+	private final IParser parser;
+
 	protected final Log log = LogFactory.getLog(this.getClass());
 
+	public FHIRClientHelper() {
+        parser = FhirContext.forDstu3().newJsonParser();
+    }
```

Comment:
There are some issues with indent, but I believe you can handle it in your upcoming PR.